tests/nlu/test_evaluation.py

```diff
@@ -949,6 +950,25 @@ async def test_nlu_comparison(tmp_path: Path):
     # combined on the same dictionary key and cannot be plotted properly
     configs = [write_file_config(config).name, write_file_config(config).name]
 
+    # mock training
+    monkeypatch.setattr(Interpreter, "load", Mock(spec=RasaNLUInterpreter))
+    monkeypatch.setattr(sys.modules["rasa.nlu"], "train", AsyncMock())
+    monkeypatch.setattr(
```

Comment:
I like mocking the training - I'm a bit hesitant about mocking the testing though. Are we sure the `testing` is covered by other tests?

Reply:
yes, there are `test_intent_evaluation_report` and `test_intent_evaluation_report_large` so I think it's perfectly fine to mock testing for this specific test